application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -372,4 +372,37 @@ static Optional<TemporaryData> computeTemporaryData(String durationText) {
      */
     record TemporaryData(Instant expiresAt, String duration) {
     }
+
+    /**
+     * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
+     *
+     * @param action gives out a {@link ModerationAction} to be able to respond and build a message
+     *        appropriately such as {@link ModerationAction#MUTE} etc.
+     * @param message is the message that is being used to build responses, such as
+     *        {@link Guild#getName()}, and reasoning for an action.
+     * @return the appropriate advice.
+     */
+    public static String getDmAdvice(ModerationAction action, String... message) {
+        if (message.length == 3) {
+            return """
+                    Hey there, sorry to tell you but unfortunately you have been %s %s from the server %s.
+                    To get in touch with a moderator, you can simply use the **/modmail** command here in this chat. Your message will then be forwarded and a moderator will get back to you soon :thumbsup:
+                    The reason for being %s is: %s
+                    """
+                .formatted(action.getVerb(), message[0], message[1], action.getVerb(), message[2]);
+        } else if (ModerationAction.UNMUTE.getVerb().equals(action.getVerb())
+                || ModerationAction.UNQUARANTINE.getVerb().equals(action.getVerb())) {
+            return """
+                    Hey there, you have been %s in the server %s.
+                    This means you can now interact with others in the server again.
+                    The reason for being %s is: %s
+                    """.formatted(action.getVerb(), message[0], action.getVerb(), message[1]);
+        }
+        return """
+                Hey there, sorry to tell you but unfortunately you have been %s, which came from the server %s.
```

Comment:
"which came from the server". where is that from? none of the original commands had such a text. please make sure u dont change the original text

Reply:
```
Hey there, sorry to tell you but unfortunately you have been muted %s in the server %s.
                        This means you can no longer send any messages in the server until you have been unmuted again.
                        If you think this was a mistake, please contact a moderator or admin of the server.
                        The reason for the mute is: %s
```

You told me to change the text, just made it sound better 👍

Reply:
i was just referring to the first sentence, which doesnt sound correct. like _"you have been muted, which came from the server Together Java"_ sounds weird. and _"you have been muted in the server Together Java"_ sounded better.

Reply:
Done